testsuite/tests/typing-misc/records.ml

```diff
@@ -268,7 +268,7 @@ Line 1, characters 0-30:
 1 | type perm = d = {y:int; x:int}
     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 Error: This variant or record definition does not match that of type "d"
-       Fields "x" and "y" have been swapped.
+       Field "y" has been moved from position 1 to 2.
```

Comment:
I think that this is a regression.

Reply:
This was a corner case because the existing weight considered that a move of one field 
```
a   b   a
↓-------↑
```
(aka a delete + insertion ) was a good as an exchange of two fields
```
a  b
↓  ↓
b  a
```
(aka two change) I have fixed this case by  tweaking the weights to prefer a swap.